README.md

```diff
@@ -1,36 +1 @@
-Simple Modular Patterns and Testing
-======================================
-To complete this assignment:
-  * fork this repository (the sub module for this specific assignment)
-  * clone down your fork
-  * place all of your work in a folder that is your full name, use `_`s instead of spaces
-  * push back up to your fork
-  * create a pull request back to the original repo
-  * submit a link to the PR in canvas
-
-Assignment Description
-----------------------------------
-This assignment will have you create a simple Javascript object that will be exported using the Node modular pattern we went over in class.
-
-Your object should have a function named 'greet' that takes a name as a parameter and returns the string 'hello ' + name 
-
-You should have at least one test that verifies the output of the function.
-
-Your submission should be a link to a pull request to your own repository.
-
-Bonus:
--------------
-For an extra point, create a command line utility that will be run using node greet.js 'some name' and will pass the input contained in that argument to the greet function and output the result to the screen.
-
-For a second bonus point, write a test that makes sure that the arguments are being processed.
- 
-
-Rubric:
-----------------------
-Proper Styling: 2pts
-
-Proper Submission: 2pts
-
-Mocha/Chai Test: 3pts
-
-Use of Modular Pattern/design of greet object/function: 3pts 
+This app greets someone by name and time of day!
```

Comment:
I moved your README into your subdirectory so it will stop causing merge conflicts